shard.yml

```diff
@@ -1,7 +1,7 @@
 name: webmock
 version: 0.13.0
 
-crystal: 0.26.0
+crystal: 0.36.0
```

Comment:
Can we include at least 1.0-dev here to be able to use this shard with upcoming nightly builds after 0.36?

Reply:
`crystal: >= 0.36.0, < 2.0.0` ?

I don't think we can use 1.0-dev as a restriction. It could be `>= 0.36.0, < 1.1.0`.

I am fine to republish this upon 1.0 with a `crystal: 1.0.0` that will be valid through all 1.x.